plasma_framework/docs/design/exit_bond_and_bounty_mechanics.md

```diff
@@ -28,7 +28,7 @@ For an invalid exit, a challenge transfers the whole bond to the challenger. Non
 **_On succesfull challenge:_**
 *Bond -> from `Exit Game Contract` to `Challenger`*
 
-The **Exit Processor** gets the bounty as a reward for processing an exit. The cost of processing multiple exits from the exit queue at once is less than the cost of processing them individually. For simplicity and to save gas however, the **Process Exit Bounty** amount associated with each exit is given out to the **Exit Processor**.
+The **Exit Processor** gets the bounty as a reward for processing an exit. The cost of processing multiple exits from the exit queue at once is less than the cumulative cost of processing them individually. Since, the **Process Exit Bounty** amount associated with each exit is given out, the **Exit Processor** can wait for the exit queue to stack up and then process exits together to get higher incentives.
```

Comment:
```suggestion
The **Exit Processor** gets the bounty as a reward for processing an exit. The cost of processing multiple exits from the exit queue at once is less than the cumulative cost of processing them individually. Since the **Process Exit Bounty** is given out for each exit processed, the **Exit Processor** can wait for the exit queue to stack up and then process multiple exits together to get a higher reward.
```